docs/cell_census_storage_and_release_policy.md

```diff
@@ -1,45 +1,48 @@
-`s3://cellxgene-data-public/`
+## Cell Census data storage policy
 
-All contents of a Cell Census build data MUST be deposited in a folder named `cell-census`: 
+The following S3 bucket MUST be used as the root to store Cell Census data:
 
-`./cell-census/`
+`s3://cellxgene-data-public/`
 
-This folder MUST contain all public builds and releases defined as:
+Cell Census data MUST be deposited under a folder named `cell-census` of the root S3 bucket:
+ 
+ `./cell-census`
+ 
+All data related to a Cell Census **release** MUST be deposited in a folder named with the **tag** of the release:
 
-* Census "build" is a SOMA collection / container containing cell data.
-* Census "RELEASE" is a "build" which has been officially blessed and released in a formal manner.
-* A "build" is named with a `tag`, which is a string of printable ASCII (eg, "foo", "2022-11-01", "1-rc2.9"):
-   `./cell-census/[tag]/`
-* a RELEASE is named with a tag following the conventional format `release-[counter]`:
-   `./cell-census/release-[counter]/`
+ `./cell-census/[tag]/`
 
-The Cell Census SOMA-related files MUST be deposited within the release folder on a folder named `soma`:
+The Cell Census **release** MUST be deposited in a folder named `soma`:
 
-`./cell-census/release-[counter]/soma/`
+`./cell-census/[tag]/soma/`
 
-All h5ads used to create the Cell Census MUST be copied within the release folder into a folder named `h5ads`:	
-`./cell-census/release-[counter]/h5ads/`
+All Cell Census **source h5ads** used to create a specific Cell Census **release** MUST be copied into a folder named `h5ads`:	
+`./cell-census/[tag]/h5ads/`
 
-Any dataset changes, additions, or deletions per release MUST be documented in the following human-readable changelog file name `changelog.txt`:
+## Cell Census release information `json`
 
-`./cell-census/changelog.txt`
 
-The publication date along with the full URI paths for the `soma` folder and the `h5ads` folder  for all Cell Census releases  MUST be recorded in a `json` file with the following naming convention and structure, which will be used as a machine- and human-readable directory of available census builds:
+The publication date along with the full URI paths for the `soma` folder and the `h5ads` folder for all Cell Census releases  MUST be recorded in a `json` file with the following naming convention and structure, which will be used as a machine- and human-readable directory of available census builds:
 
 
 `./cell-census/releases.json`
 
-This file MUST be in `json` formats where the parent keys are release identifiers (alias or name). The alias `"latest"` MUST be present. This `json` file MUST follow this schema:
+* This file MUST be in `json` formats where the parent keys are release identifiers (alias or name). 
+* The alias `latest` MUST be present and MUST point to the **Weekly Cell Census release**. 
+* The prefix `V` MUST be used followed by an integer counter to label long-term supported Cell Census releases, e.g. `V1`.
```

Comment:
do you want to define the version scheme?  It is certainly possible that any given LTS release (say June 2023, aka V1) might require corrections long after the date (e.g., for simple technical reasons, redaction, etc).  How is that "revision" encoded, if at all?